front/app/components/IdeaCard/index.tsx

```diff
@@ -223,7 +223,8 @@ const CompactIdeaCard = memo<Props>(
               <StyledUserName userId={authorId || null} />
               <Separator aria-hidden>&bull;</Separator>
               <TimeAgo>
-                {timeAgo(Date.parse(idea.attributes.created_at), locale)}
+                {!isNilOrError(locale) &&
+                  timeAgo(Date.parse(idea.attributes.created_at), locale)}
               </TimeAgo>
```

Comment:
Would add the check at the component level next to idea
```

    if (isNilOrError(idea) || isNilOrError(locale)) {
      return null;
    }
```

Unlikely to happen but if locale is null/error, there will be margin-right from the TimeAgo styles. So the other option would be to do the check outside of TimeAgo.